google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -221,6 +240,71 @@ interfaces:
     field_name_patterns:
       subscription: subscription
     timeout_millis: 60000
+  - name: ListSnapshots
+    flattening:
+      groups:
+      - parameters:
+        - project
+    required_fields:
+    - project
+    request_object_method: true
+    page_streaming:
+      request:
+        page_size_field: page_size
+        token_field: page_token
+      response:
+        token_field: next_page_token
+        resources_field: snapshots
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      project: project
+    timeout_millis: 60000
+  - name: CreateSnapshot
+    flattening:
+      groups:
+      - parameters:
+        - name
+        - subscription
+    required_fields:
+    - subscription
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: snapshot
+      subscription: subscription
+    timeout_millis: 60000
+  - name: DeleteSnapshot
+    flattening:
+      groups:
+      - parameters:
+        - snapshot
+    required_fields:
+    - snapshot
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      snapshot: snapshot
+    timeout_millis: 60000
+  - name: Seek
+    flattening:
+      groups:
+      - parameters:
+        - subscription
+        - time
+      - parameters:
+        - subscription
+        - snapshot
```

Comment:
This is because of the oneof containing time and snapshot. Admittedly this is what I am not 100% sure of as said in my previous comment.

I am assuming that with two flattening groups, two methods will be generated.

`Seek(String subscription, Time time)` and `Seek(String subscription, String snapshot)`